### ISSUE TITLE
Model availability

### ISSUE THREAD

**Issue by dafnevk** [guyrosin/temporal_attention#1]
Thanks for making this code available, I read your paper with great interest.

Is your trained model that you describe in the paper publicly available? I cannot find a link to it anywhere in your paper or repository.

**Comment by guyrosin** [comment]
Hi @dafnevk, thank you for your interest in our paper!
Please find a trained model for the SemEval-English corpus here: https://karpef.cs.technion.ac.il/index.php/s/GJ8ETxbq9ZkGffr

**Comment by dafnevk** [comment]
Thanks for sharing the model!
However, I have trouble getting it to give meaningful outputs and to reproduce the results from your paper.
See [this notebook](https://github.com/Semantics-of-Sustainability/temporal_attention/blob/main/TryModel.ipynb) where I try it out for mask filling, and [this notebook](https://github.com/Semantics-of-Sustainability/temporal_attention/blob/main/Reproduce_semantic_change_plot.ipynb) plotting the outputs of the [semantic_change_detection.py](https://github.com/Semantics-of-Sustainability/temporal_attention/blob/main/semantic_change_detection.py) script.
Do you have any idea what could be going wrong?

And it is it also possible for you to release the TempoBERT model?

**Comment by guyrosin** [comment]
Hi @dafnevk, 
I'm sorry the model didn't work as expected! I've already graduated so I don't have access to the server I used for experiments, but I'll do my best to try figure out what went wrong.

In the meanwhile, here are some trained TempoBERT models:
LiverpoolFC: https://karpef.cs.technion.ac.il/index.php/s/zQATC4Y2T8aipHN
SemEval English: https://karpef.cs.technion.ac.il/index.php/s/5XYW7GW9AbPXXZP
SemEval Latin: https://karpef.cs.technion.ac.il/index.php/s/yRRGGwMd6dA7qtH
If I'm not mistaken, the SemEval models work best when using just the last hidden layer to create time-specific word embeddings, and the LiverpoolFC model works best using the last 4 layers.
